src/pages/[platform]/build-a-backend/auth/connect-your-frontend/using-the-authenticator/index.mdx

```diff
@@ -0,0 +1,66 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Using the Authenticator',
+  description: 'Learn how to use the Authenticator connected component from the Amplify UI library',
+  platforms: [
+    // 'android',
```

Comment:
unused platforms

Reply:
filed an issue to port in mobile examples